samples/dashboards/demo/datagrid-mathmodifier/demo.js

```diff
@@ -67,15 +68,40 @@ Dashboards.board('container', {
                 chart: {
                     animation: false,
                     type: 'line',
-                    zooming: false
+                    zooming: false,
+                    events: {
+                        redraw: function () {
+                            if (!this.series[1].options.yAxis) {
+                                this.series[1].update({
+                                    yAxis: 1
+                                });
+                            }
+                        }
+                    }
```

Comment:
What is the purpose of this callback? Is it needed for the demonstration?

Reply:
It's the only possible workaround to assign a series to another axis.